pages/index.tsx

```diff
@@ -1,14 +1,17 @@
+import { TriangleDownIcon } from '@chakra-ui/icons'
```

Comment:
Instead of using Chakra's icons, could we use something from [`react-feather`](https://feathericons.com/)? The package is already installed and more comprehensive than Chakra's.

Once we switch all icons to Feather, we should be able to uninstall the Chakra icons package.

Reply:
switched it to use chevrons up and down depending on whether list is scending or not